starwars/src/components/Data.js

```diff
@@ -34,6 +36,8 @@ const Data = () => {
                     </div>
                 )
             })}
+            <Button content="First Page" onClick={() => setUrl("https://swapi.co/api/people/?format=json")}></Button>
+            <Button secondary onClick={() => setUrl("https://swapi.co/api/people/?page=2&format=json")}>Next Page</Button>
```

Comment:
did you get the extra pages to work properly when it gets to 9, i remember we were working on that